src/modules/helpers/index.ts

```diff
@@ -158,6 +158,43 @@ function legacyRegexpStringParse(
   return string;
 }
 
+/**
+ * Parses the given string symbol by symbol and replaces the placeholders with digits (`0` - `9`).
+ * `!` will be replaced by digits >=2 (`2` - `9`).
+ *
+ * @internal
+ *
+ * @param faker Faker instance
```

Comment:
```suggestion
 * @param faker A Faker instance.
```

Same as the method above.